common/src/main/java/fr/rakambda/fallingtree/common/tree/BreakAbortionCause.java

```diff
@@ -0,0 +1,22 @@
+package fr.rakambda.fallingtree.common.tree;
+
+public enum BreakAbortionCause implements IBreakAttemptResult {
```

Comment:
As this is the result itself, maybe it can be renamde to something like `AbortedResult`